bedrock/firefox/templates/firefox/geolocation.html

```diff
@@ -21,12 +21,21 @@
 <div id="main-feature">
   <h1>{{ _('Location-Aware Browsing') }}</h1>
   <p>
-  {% trans url='#geo-demo', id='try-geolocation', style='display:none' %}
-    Firefox can tell websites where you’re located so you can find info
-    that’s more relevant and more useful. It’s about making the Web smarter
-    &ndash; and is done in a way that totally respects your privacy.
-    <a href="{{ url }}" id="{{ id }}" style="{{ style }}">Give it a try!</a>
-  {% endtrans %}
+    {% if l10n_has_tag('remove-inline-styles') %}
```

Comment:
The only change is the removal of `style="{{ style }}"`, right? I think I can script that, without using a new l10n tag.